### DISCUSSION TITLE
colouring tab titlebar

### DISCUSSION THREAD

**Discussion by kraftwerk28** [i3/i3#4574] [Q&A]
So I have the following layout:
![image](https://user-images.githubusercontent.com/31807671/136657527-6929cf9a-62dc-471f-85a2-98ecef20d294.png)
```
    tabbed layout
         / \
Alacritty   H[Alacritty Alacritty]
                 /      \
         Alacritty       Alacritty
```

I want to make `H[Alacritty Alacritty]` title bar to be coloured in same way as it is in focused tab (e.g. if I'm inside child node of the tab, colour it in `client.focused`)
My config for colors:
```
client.focused          "#ff8800" "#884400" "#ffffff" "#ff4400" "#ff8800"
client.focused_inactive "#666666" "#222222" "#ffffff" "#444444" "#666666"
client.unfocused        "#666666" "#222222" "#ffffff" "#444444" "#666666"
```

One suggested me to make `client.focused` and `client.unfocused` have the same colors, but that doesn't work, because it also highlights most recent focused window in orange.
I hope my explanation was clear, thank you in advance.

**Comment by Airblader** [comment]
These are the only options we expose. There are no other ways to color the decorations.

**Reply by kraftwerk28** [reply]
Are there plans to add such field to `client.`? In my opinion it will make navigation more clear, i.e. if I have, say, 5 tabs with terminals and then inside one tab I create a split, the current tab becomes non highlighted, which is a bit harder to read
![image](https://user-images.githubusercontent.com/31807671/136658641-33f66edb-c088-4004-9f6c-6b9ad67117aa.png)

**Reply by Airblader** [reply]
There are no concrete plans for any changes, no. I think an argument could potentially be made that this is a bug, but I'd need to think about it some more. Feel free to report it as a bug for now, though. Maybe @orestisfl has an opinion here, too.

**Comment by orestisfl** [comment]
IIUC, this color is controlled by `client.focused_inactive`

![image](https://user-images.githubusercontent.com/5778622/136996045-2c2db8f8-a49a-4851-95ab-fc32336a3ddb.png)


```
client.focused          "#ff8800" "#884400" "#ffffff" "#ff4400" "#ff8800"
client.focused_inactive          "#ff8800" "#884400" "#ffffff" "#ff4400" "#ff8800"
client.unfocused        "#666666" "#222222" "#ffffff" "#444444" "#666666"
```

Though it will also influence the colors of the second screen if you have one :/

**Reply by orestisfl** [reply]
Nevermind, just saw your comment in https://github.com/i3/i3/issues/4575

> Such behavior is not achievable by using client.focused_inactive, because that would also highlight last focused split (not necessarily tab title)
